.travis.yml

```diff
@@ -35,6 +35,7 @@ install:
   - pip install unidecode
   - pip install pycountry
   - pip install suds-jurko==0.6
+  - pip install https://github.com/damdam-s/roulier/archive/workarounds.zip
```

Comment:
you can also use the following : 
` - pip install git+https://github.com/akretion/roulier.git `

https://github.com/akretion/carrier-delivery/blob/8-roulier-travis/.travis.yml#L37